packages/app-store/routing-forms/components/FormActions.tsx

```diff
@@ -486,3 +514,7 @@ export const FormAction = forwardRef(function FormAction<T extends typeof Button
     </DropdownMenuItem>
   );
 });
+
+export const createAction = ({ router, teamId }: { router: NextRouter; teamId: number | null }) => {
```

Comment:
Exposing this createAction outside of FormActions so that `CreateButton` can use it.